rules_gapic/java/resources/gradle/client.gradle.tmpl

```diff
@@ -7,8 +7,8 @@ buildscript {
 apply plugin: 'java'
 
 description = 'GAPIC library for {{name}}'
-group = '{{group}}'
-version = '{{version}}'
+group = 'com.google.cloud'
```

Comment:
I might be missing something in how this is used but hard coding "cloud" into the gradle template seems off to me. Why hard code it?

Reply:
Yes, this is absolutely off (wrong). Unfortunately we already do it even deeper (inside the generator itself, when we generate same files via artman+gapic-generator bundle). For java it is here: [build_base.gradle.snip](https://github.com/googleapis/gapic-generator/blob/master/src/main/resources/com/google/api/codegen/metadatagen/java/grpc/build_base.gradle.snip#L11). 

Gapic-generator is contaminated with stuff like this, for example python setup snip hardcodes google-cloud-python in it: [setup.py.snip](https://github.com/googleapis/gapic-generator/blob/master/src/main/resources/com/google/api/codegen/py/setup.py.snip#L60). Other languages have similar stuff.